From 16dc683aa50be9789d1674734b06a8a955ff22ad Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 26 Nov 2015 01:36:40 -0800
Subject: [PATCH] lib/rpmfc.c: fix for N32 MIPS64

It is 'N32 MIPS64', not 'N32 MIPS32' as command file shows:
$ file image/usr/bin/getent
getent: ELF 32-bit MSB executable, MIPS, N32 MIPS64 [snip]

And "rpm -qp --filecolor" was wrong (it was 1, but should be 4).

Upstream-Status: Submitted [RPM5 maintainer]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 lib/rpmfc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/rpmfc.c b/lib/rpmfc.c
index fde00c7..49779f8 100644
--- a/lib/rpmfc.c
+++ b/lib/rpmfc.c
@@ -575,7 +575,7 @@ static struct rpmfcTokens_s rpmfcTokens[] = {
   { " not stripped",		RPMFC_NOTSTRIPPED },
   { " archive",			RPMFC_ARCHIVE },
 
-  { "MIPS, N32 MIPS32",		RPMFC_ELFMIPSN32|RPMFC_INCLUDE },
+  { "MIPS, N32 MIPS64",		RPMFC_ELFMIPSN32|RPMFC_INCLUDE },
   { "ELF 32-bit",		RPMFC_ELF32|RPMFC_INCLUDE },
   { "ELF 64-bit",		RPMFC_ELF64|RPMFC_INCLUDE },
 
-- 
1.7.9.5

